packages/gatsby/src/redux/reducers/static-queries-by-template.ts

```diff
@@ -0,0 +1,22 @@
+import { ActionsUnion, IGatsbyState } from "../types"
+
+export const staticQueriesByTemplateReducer = (
+  state: IGatsbyState["staticQueriesByTemplate"] = new Map(),
+  action: ActionsUnion
+): IGatsbyState["staticQueriesByTemplate"] => {
+  switch (action.type) {
+    case `REMOVE_TEMPLATE_COMPONENT`:
```

Comment:
Keeping things consistent
```suggestion
    case `REMOVE_STATIC_QUERIES_BY_TEMPLATE`:
```